applications/data_analysis/climatology.ipynb

```diff
@@ -0,0 +1,3551 @@
+{
```

Comment:
<p>(from convo) Above this section, add a "data" section where you introduce the data, explain how to get it, and output it for reference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>